### ISSUE TITLE
Code should follow PEP8 guidelines

### ISSUE THREAD

**Issue by jadhavganesh** [django-fluent/django-fluent-dashboard#17]
Following errors are found when flake8 is run,
branch: master
$ pip install flake8
$ flake8 --ignore=E501 .
./setup.py:48:19: E251 unexpected spaces around keyword / parameter equals
./setup.py:48:21: E251 unexpected spaces around keyword / parameter equals
./docs/conf.py:14:11: E401 multiple imports on one line
./docs/conf.py:26:1: E265 block comment should start with '# '
./docs/conf.py:42:1: E265 block comment should start with '# '
./docs/conf.py:62:1: E265 block comment should start with '# '
./docs/conf.py:66:1: E265 block comment should start with '# '
./docs/conf.py:68:1: E265 block comment should start with '# '
./docs/conf.py:75:1: E265 block comment should start with '# '
./docs/conf.py:78:1: E265 block comment should start with '# '
./docs/conf.py:82:1: E265 block comment should start with '# '
./docs/conf.py:86:1: E265 block comment should start with '# '
./docs/conf.py:92:1: E265 block comment should start with '# '
./docs/conf.py:104:1: E265 block comment should start with '# '
./docs/conf.py:107:1: E265 block comment should start with '# '
./docs/conf.py:111:1: E265 block comment should start with '# '
./docs/conf.py:114:1: E265 block comment should start with '# '
./docs/conf.py:118:1: E265 block comment should start with '# '
./docs/conf.py:123:1: E265 block comment should start with '# '
./docs/conf.py:132:1: E265 block comment should start with '# '
./docs/conf.py:136:1: E265 block comment should start with '# '
./docs/conf.py:139:1: E265 block comment should start with '# '
./docs/conf.py:143:1: E265 block comment should start with '# '
./docs/conf.py:146:1: E265 block comment should start with '# '
./docs/conf.py:149:1: E265 block comment should start with '# '
./docs/conf.py:152:1: E265 block comment should start with '# '
./docs/conf.py:155:1: E265 block comment should start with '# '
./docs/conf.py:158:1: E265 block comment should start with '# '
./docs/conf.py:161:1: E265 block comment should start with '# '
./docs/conf.py:166:1: E265 block comment should start with '# '
./docs/conf.py:169:1: E265 block comment should start with '# '
./docs/conf.py:178:1: E122 continuation line missing indentation or outdented
./docs/conf.py:179:1: E122 continuation line missing indentation or outdented
./docs/conf.py:179:1: E265 block comment should start with '# '
./docs/conf.py:181:1: E122 continuation line missing indentation or outdented
./docs/conf.py:182:1: E122 continuation line missing indentation or outdented
./docs/conf.py:182:1: E265 block comment should start with '# '
./docs/conf.py:184:1: E122 continuation line missing indentation or outdented
./docs/conf.py:185:1: E122 continuation line missing indentation or outdented
./docs/conf.py:185:1: E265 block comment should start with '# '
./docs/conf.py:191:3: E121 continuation line under-indented for hanging indent
./docs/conf.py:197:1: E265 block comment should start with '# '
./docs/conf.py:201:1: E265 block comment should start with '# '
./docs/conf.py:204:1: E265 block comment should start with '# '
./docs/conf.py:207:1: E265 block comment should start with '# '
./docs/conf.py:210:1: E265 block comment should start with '# '
./docs/conf.py:213:1: E265 block comment should start with '# '
./docs/conf.py:226:1: E265 block comment should start with '# '
./docs/conf.py:235:3: E121 continuation line under-indented for hanging indent
./docs/conf.py:241:1: E265 block comment should start with '# '
./docs/conf.py:244:1: E265 block comment should start with '# '
./docs/conf.py:247:1: E265 block comment should start with '# '
./fluent_dashboard/items.py:4:1: F401 'django' imported but unused
./fluent_dashboard/items.py:34:14: E201 whitespace after '{'
./fluent_dashboard/items.py:38:13: E124 closing bracket does not match visual indentation
./fluent_dashboard/items.py:92:69: F841 local variable 'e' is assigned to but never used
./fluent_dashboard/appsettings.py:50:21: E241 multiple spaces after ':'
./fluent_dashboard/appsettings.py:52:49: E261 at least two spaces before inline comment
./fluent_dashboard/appsettings.py:52:50: E262 inline comment should start with '# '
./fluent_dashboard/appsettings.py:59:62: E261 at least two spaces before inline comment
./fluent_dashboard/appsettings.py:59:63: E262 inline comment should start with '# '
./fluent_dashboard/appsettings.py:79:65: E261 at least two spaces before inline comment
./fluent_dashboard/appsettings.py:79:66: E262 inline comment should start with '# '
./fluent_dashboard/appsettings.py:179:5: E265 block comment should start with '# '
./fluent_dashboard/dashboard.py:14:1: F401 'admin_tools.dashboard.modules.Group' imported but unused
./fluent_dashboard/dashboard.py:91:61: E262 inline comment should start with '# '
./fluent_dashboard/dashboard.py:119:12: E713 test for membership should be 'not in'
./fluent_dashboard/modules.py:188:9: E122 continuation line missing indentation or outdented
./fluent_dashboard/modules.py:189:9: E122 continuation line missing indentation or outdented
./fluent_dashboard/modules.py:216:17: E221 multiple spaces before operator

**Comment by vdboor** [comment]
agreed! Most changes are merged now, only line length should be fixed (and `conf.py`, but the project code is more important)
